cpp/lib/src/IniFile.cc

```diff
@@ -585,13 +585,13 @@ void IniFile::processFile(const std::string &external_filename) {
             std::string buf;
             std::getline(ini_file, buf);
             ++getCurrentLineNo();
-            line += buf;
+            line += StringUtil::Trim(buf, " \t");
```

Comment:
Why not use TrimWhite?

Reply:
The `StringUtil::Trim()` approach is used in numerous positions in `IniFile.cc`. Use TrimWhite() anyway?

Reply:
Fair enough!